modules/tinymce/src/plugins/table/test/ts/browser/ui/TableRowHeaderUiTest.ts

```diff
@@ -0,0 +1,106 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyHooks, TinySelections } from '@ephox/mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/table/Plugin';
+import Theme from 'tinymce/themes/silver/Theme';
+
+import { clickOnButton, makeCell, pClickOnMenuItem } from '../../module/test/TableModifiersTestUtils';
+
+describe('browser.tinymce.plugins.table.ui.TableRowHeaderUiTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    plugins: 'table',
+    base_url: '/project/tinymce/js/tinymce',
+    menu: {
+      table: { title: 'Table', items: 'tablerowheader' },
+    },
+    menubar: 'table',
+    toolbar: 'tablerowheader',
+    indent: false,
+  }, [ Plugin, Theme ], true);
+
+  const setEditorContentAndSelection = (editor: Editor, headBody: string) => {
```

Comment:
Can we type `headBody`? I also don't think we need to make it confusing by leaving the 't' out.

maybe `tableSection: 'thead' | 'tbody'`

Reply:
I did actually have it typed, but I wasn't sure people would like it, so I put it as a string. My bad.